src/test/addresses_goerli.sol

```diff
@@ -346,6 +346,16 @@ contract Addresses {
         addr["RWA013_A_URN"]                    = 0xdC47d203753D3B5fb4fcD5900EBd96b0eC6761B6;
         addr["RWA013_A_OUTPUT_CONDUIT"]         = 0xc5A1418aC32B5f978460f1211B76B5D44e69B530;
         addr["RWA013_A_INPUT_CONDUIT"]          = 0xc5A1418aC32B5f978460f1211B76B5D44e69B530;
+        addr["RWA014"]                          = 0x22a7440DCfF0E8881Ec93cE519c34C15feB2A09a;
+        addr["PIP_RWA014"]                      = 0x0dC2eeaAbD3c8F6fD7FB62D690AfEEA8c7AE5A6F;
```

Comment:
If we know these it needs to match the sheet. We can discuss in a call.

Reply:
Confirmed this on the call, I get `0x0dC2eeaAbD3c8F6fD7FB62D690AfEEA8c7AE5A6F` as well when I run `cast compute-address 362dfe51e4f91a8257b8276435792095ee5d85c3`